source/jormungandr/tests/schema_tests.py

```diff
@@ -374,3 +376,21 @@ def test_stop_schedules(self):
 
         # we check that the response is not empty
         assert any((o.get('date_times') for o in obj.get('stop_schedules', [])))
+
+
+@dataset({"main_routing_test": {}})
+class TestSwaggerSchemaBssStands(AbstractTestFixture, SchemaChecker):
+    def test_pois_with_stands_on_first_poi(self):
+        with mock_bss_providers(pois_supported=[]):
+            self._check_schema(
+                '/v1/coverage/main_routing_test/pois/poi:station_1'
```

Comment:
Do you think this could be generalised to all (most) endpoints ? 
Maybe based on what we did here: https://github.com/CanalTP/navitia/blob/55a22f778cd88b5d0633549105a403cb8317670e/source/jormungandr/tests/schema_tests.py#L325-L346.